tests/unittests/ImporterTestUtils.h

```diff
@@ -48,4 +48,13 @@ unsigned countNodeKind(Function *F, Kinded::Kind kind) {
   return count;
 }
 
+/// Helper function to get the save node from a Variable \p var.
+SaveNode *getSaveNodeFromVariable(Variable *var) {
+  auto &varUsers = var->getUsers();
+  assert(varUsers.size() == 1);
```

Comment:
Add a \pre saying that var should have only one user.

Reply:
I already had merged to unbreak quickly. Sent a follow up PR https://github.com/pytorch/glow/pull/1515